projects/openapi-utilities/src/index.ts

```diff
@@ -1,6 +1,8 @@
 import { OpenAPITraverser } from './openapi3/implementations/openapi3/openapi-traverser';
 import { OpenAPIV3 } from 'openapi-types';
 import { factsToChangelog } from './openapi3/sdk/facts-to-changelog';
+export { DereferencedOpenAPIV2, DereferencedOpenAPIV3, DereferencedOpenAPIV3_1 } from './flat-openapi-types'
```

Comment:
Do we like these names or do we want something shorter? 

`FlatOpenAPI`? `OAS_3_1`?

Reply:
`DerefOpenAPI`?

Reply:
went with Flat